spec/lib/hackney/service_charge/universal_housing_lease_gateway_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+describe Hackney::ServiceCharge::UniversalHousingLeaseGateway, universal: true do
+  context 'when retrieving tenancy refs for cases in arrears' do
+    subject { described_class.lease_in_arrears }
+
+    let(:lease_rent_group) { 'LSC' }
+
+    context 'when there are no tenancies' do
+      it 'returns none' do
+        expect(subject).to be_empty
+      end
+    end
+
+    context 'when there is one tenancy in arrears and in the leasehold rent group' do
+      before { create_uh_tenancy_agreement(tenancy_ref: '000001/01   ', current_balance: 50.00, rentgrp_ref: lease_rent_group) }
+
+      it 'returns that tenancy with stripped whitespace' do
+        expect(subject).to eq(['000001/01'])
+      end
+    end
+
+    context 'when there are three tenancies in arrears, but only one is in the leasehold rent group' do
+      before do
+        create_uh_tenancy_agreement(tenancy_ref: '000001/01', current_balance: 50.00, rentgrp_ref: lease_rent_group)
+        create_uh_tenancy_agreement(tenancy_ref: '000002/01', current_balance: 50.00)
+        create_uh_tenancy_agreement(tenancy_ref: '000003/01', current_balance: 50.00)
+      end
+
+      it 'returns only the master account tenancy' do
```

Comment:
I think you meant to 
```suggestion
      it 'returns only the one in lease rent group' do
```